main/Accordion/index.tsx

```diff
@@ -1,196 +1,98 @@
-import {
-  Animated,
-  InteractionManager,
-  LayoutChangeEvent,
-  StyleProp,
-  StyleSheet,
-  TouchableOpacity,
-  View,
-  ViewStyle,
-} from 'react-native';
-import React, { ReactElement, useEffect, useState } from 'react';
+import { Animated, LayoutChangeEvent, View } from 'react-native';
+import React, { FC, ReactElement, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components/native';
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    flexDirection: 'column',
-    overflow: 'hidden',
-  },
-  underline: {
-    width: '100%',
-    height: 1,
-    position: 'absolute',
-    top: 0,
-  },
-  content: {
-    flex: 1,
-    flexDirection: 'column',
-    justifyContent: 'center',
-    alignItems: 'flex-start',
-  },
-  contentChild: {
-    padding: 12,
-  },
-  contentView: {
-    flexDirection: 'row',
-    alignItems: 'center',
-    height: '100%',
-  },
-  contentTxt: {
-    color: 'black',
-    marginLeft: 8,
-    fontSize: 12,
-  },
-  contentFooter: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'flex-start',
-    alignItems: 'center',
-    height: 48,
-    paddingHorizontal: 12,
-  },
-});
+const Container = styled.TouchableWithoutFeedback`
+  width: 100%;
+`;
+const HeaderContainer = styled(Container)`
+  flex-direction: row;
+  justify-content: space-between;
+`;
+const ContentContainer = styled(Container)``;
 
 interface Props {
-  contentVisibleOnLoad?: boolean;
-  backgroundColor?: string;
-  titleBackground?: string;
-  contentBackground?: string;
-  underlineColor?: string;
+  contentVisible?: boolean;
+  header: ReactElement;
+  children: ReactElement;
   visibleElement?: ReactElement;
   invisibleElement?: ReactElement;
-  header: React.ReactElement;
-  style?: StyleProp<ViewStyle>;
-  children: React.ReactElement;
+  isAnimated?: boolean;
+  duration?: number;
 }
 
-interface ContentVisibleState {
-  isContentVisible: boolean;
-  currentAnimation: AnimationType;
-}
-
-enum AnimationType {
-  showContent = 'showContent',
-  hideContent = 'hideContent',
-  null = 'null',
-}
+type EVProps = {
```

Comment:
What does `EV` stands for?

Reply:
It means short for `ElementValueProps`.
It should be changed.